packages/kit/src/SelectField.tsx

```diff
@@ -10,7 +9,7 @@ const SelectFieldProps = z.object({
   id: z.string().optional(),
   label: z.string().optional(),
   unselectedLabel: z.string().optional().default('...'),
-  onValueChange: EventDataStateSchema.optional(),
+  onValueChange: z.function().args(z.string()).optional(),
```

Comment:
can zod validate fn arguments? how does that work? I'm guessing this is just for TS annotation?

Reply:
yeah, TS annotation + if I were to guess, typeof check. Nothing more. And there's no way to check for a React component type, but you can use something such as `z.custom(ReactNode)`, which offers TS, but no validation.